### PR TITLE
Header

### PR DESCRIPTION
**Pull request by sgiraldo112**
Added global ```header.jsx``` and ```footer.jax``` and added component to active pages

### CODE REVIEW

**Comment by Bakushin10** [comment]
you still hvae merge conflict on ```Professor.jsx ``` and ```ProfessorDetails.jsx```

File: client/components/Header.jsx [hunk 1]
```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+import { Navbar, Nav, NavItem, MenuItem, NavDropdown } from 'react-bootstrap';
+
+const Header = () => (
+  <div className="container">
+    <link rel="stylesheet" href="./css/Header.css" />
```
Comment (Bakushin10) [inline client/components/Header.jsx]: when you are loading css in react, it should be ```import './css/Header.css'``` . and you dont define it inside of class
https://stackoverflow.com/questions/39853646/how-to-import-a-css-file-in-a-react-component

File: client/components/Header.jsx [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+    <link href="https://fonts.googleapis.com/css?family=Lobster|Pacifico" rel="stylesheet" />
```
Comment (Bakushin10) [inline client/components/Header.jsx]: I am not 100% sure if you can import api like this but I could be wrong

**Comment by sgiraldo112** [comment]
Made changes to import ```header.css``` from ```index.ejs```. View is still wrong on Firefox, will continue to work on that, @TDinaso559 please try to run on chrome and tell me what you see

**Comment by Bakushin10** [comment]
it seems like the problem is particular react-bootstrap version has a bug on ```MenuItem``` components.

File: client/components/Header.jsx (hunk 2 above)
Comment (Bakushin10) [reply client/components/Header.jsx]: moved the line of code to ```heder.css```
the problem has been fixed

**Comment by TDinaso559** [comment]
Just reviewed it and I think it is what @Bakushin10 was saying.

**Comment by Bakushin10** [comment]
The issue of the header is taking longer time than I expected. I am closing this PR. @sgiraldo112 will work on header/footer in different branch

**Pull request closed without merging**